### DISCUSSION TITLE
https://lotu.sh/en+api-methods loading

### DISCUSSION THREAD

**Discussion by dd45e640b42e6da7da96faee3996ef7c** [filecoin-project/lotus#3637] [Storage Provider] [answered]
**Describe the bug**
When loading [https://lotu.sh/en+api-scripting-support](https://lotu.sh/en+api-scripting-support) firefox starts lagging and shows the "script slows down browser" warning.

**To Reproduce**
1. Open the mentioned page

**Expected behavior**
Open without errors/warnings.

**Screenshots**
![2020-09-08_09-13](https://user-images.githubusercontent.com/60559388/92423413-a18c7c00-f1b3-11ea-8e2f-06623279cee0.png)

**Version (run `lotus version`):**
not lotus code related
**Additional context**
The page has no need to load dynamically i think. It's plain text build on a specific commit of the master branch i believe.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
https://filecoinproject.slack.com/archives/CQLDFHAJY/p1599526868008900

![2020-09-08_16-03](https://user-images.githubusercontent.com/60559388/92449628-e8e32e80-f1ec-11ea-946e-b26ae5336e45.png)

https://bafybeigvaalsxlttynfkjprjbnnhxkoxlybt2av56br5z4sto5pj2ynkei.ipfs.dweb.link/en+api

**Comment by jennijuju** [comment]
Doesnt seem like lotus related but can be an interesting discussion topic, converting!

**Comment by SgtCoin** [comment, marked as answer]
Page no longer exists, archiving for future reference.
